File: sdk/async-scala/src/main/scala/io/swagger/client/model/ApprovalRule.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class ApprovalRule (
  ID: Option[String],
Name: Option[String],
Description: Option[String],
ApprovingGroupID: Option[String],
RuleExpression: Option[String],
Scope: Option[String],
ScopeTimeUnit: Option[String],
ScopeTimeNumber: Option[Integer],
ScopeStartDate: Option[Date],
ExpireAfterTimeUnit: Option[String],
ExpireAfterNumber: Option[Integer],
ApproveOnExpire: Option[Boolean],
xp: Option[Any]
)

File: sdk/slim/lib/Models/ApprovalRule.php
<?php
/*
 * ApprovalRule
 */
namespace \Models;

/*
 * ApprovalRule
 */
class ApprovalRule {
    /* @var string $iD  */
    private $iD;
/* @var string $name  */
    private $name;
/* @var string $description  */
    private $description;
/* @var string $approvingGroupID  */
    private $approvingGroupID;
/* @var string $ruleExpression  */
    private $ruleExpression;
/* @var string $scope  */
    private $scope;
/* @var string $scopeTimeUnit  */
    private $scopeTimeUnit;
/* @var int $scopeTimeNumber  */
    private $scopeTimeNumber;
/* @var \DateTime $scopeStartDate  */
    private $scopeStartDate;
/* @var string $expireAfterTimeUnit  */
    private $expireAfterTimeUnit;
/* @var int $expireAfterNumber  */
    private $expireAfterNumber;
/* @var Bool $approveOnExpire  */
    private $approveOnExpire;
/* @var object $xp  */
    private $xp;
}
